mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -96,34 +101,69 @@ def mjcf_geom_to_sdf(body):
 
     link.set_raw_pose(su.get_pose_from_mjcf(body))
 
+    def get_orientation(geom):
+        """
+        Get orientation from a MJCF geom when it's defined with "fromto"
+        """
+        if geom.fromto is not None:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
+            v2 = Vector3d(geom.fromto[3], geom.fromto[4], geom.fromto[5])
+            vec = (v1 - v2).normalize()
+            z = Vector3d(0, 0, 1)
+            q_real = z.cross(vec)
+            s = q_real.length()
+            quat = Quaterniond()
+            if s < 1E-10:
+                quat.set_x(1)
+                quat.set_y(0)
+                quat.set_z(0)
+            ang = math.atan2(s, vec.z())
+            quat.set_w(math.cos(ang / 2.0))
+            quat.set_x(q_real.x() * math.sin(ang / 2.0))
+            quat.set_y(q_real.y() * math.sin(ang / 2.0))
+            quat.set_z(q_real.z() * math.sin(ang / 2.0))
+            return quat.normalized()
```

Comment:
This might be correct, but gz-math already has functionality for this:
```suggestion
            quat = Quaterniond()
            quat.set_from_2_axes(z, vec)
            return quat
```

Reply:
https://github.com/gazebosim/gz-mujoco/pull/47/commits/fec6e81a1f10b25c1953c1a0ce844b5dc368de18